tests/unit/core/components/TestAMInstallButton.js

```diff
@@ -525,31 +524,31 @@ describe(__filename, () => {
     expect(root.find('.AMInstallButton-button')).toHaveLength(0);
 
     expect(root.find('.AMInstallButton-loading')).toHaveLength(1);
-    expect(root.find(AnimatedIcon)).toHaveLength(1);
+    expect(root.find(Icon)).toHaveLength(1);
```

Comment:
should we add this here:

`expect(root.find(Icon)).toHaveProp('name', 'loading');`

to show the Icon is in fact loading?